docs/Install_System_Images_of_Android.md

```diff
@@ -0,0 +1,77 @@
+
+##   
+Install system-images of additional Android versions
+
+**1. Listing Available SDK Packages:**
+
+```
+sdkmanager --list
```

Comment:
`sdkmanager` would not work directly on majority of systems. Also mention the steps on how to go to the location where `sdkmanager` binary is present and how to run it. Or maybe we need to add the binaries location to PATH. The same goes for `avdmanager`.

Also, on Windows, I think we need to use `sdkmanager.exe` to run it, please verify the same.